daemon/list.go

```diff
@@ -668,3 +685,56 @@ func populateImageFilterByParents(ancestorMap map[image.ID]bool, imageID image.I
 		ancestorMap[imageID] = true
 	}
 }
+
+// Events returns events generated by daemon
```

Comment:
I understand it goes in there; may need a different phrasing to describe where it's written to